### PR TITLE
fix: various storybook improvements

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Various storybook improvements

**Solving issue**  

1. sort stories alphabetically
2. add Intro/Foundations/Utilities,Components/Patterns categories in Storybook
3. add Scania theme (to be checked with @alexandereneroth )
4. add font-size: 14px back to reset file
5. remove demo styling on Link component
6. add "Measure" addon for Storybook
7. add "Tegel" webiste under package homepage row
8. add "grid" viewports values to Storybook addon
9. set layout: padded as default in Storybook 
10. remove demo wrapper in footer component
11. add white,grey and black background to the Storybook
12. layout: centered for Badges/Chips/Datetime/Link/Toast/Toggle
13. Toast and Card to stories to have TS extension 
14. add PNG file extension to the global Typescript library
15. simplify Footer story
16. adding manager-head file for SEO needs
17.  minimising console output when running "npm run tegel"
18.  add "quiet" flag in storybook build script
19. commenting out code-sample component due to build issues

**How to test**  
1. Try to run Storybook as usual, there should be no error

**Additional context**  
Discuss with Alex about storybook dark mode and theming packages.

### CODE REVIEW

File: tegel/src/components/footer/footer.stories.ts [hunk 1]
```diff
@@ -81,13 +81,10 @@ const Template = ({ topPart = false }) =>
...
   `,
   );
 
 export const Default = Template.bind({});
-
-export const TopPart = Template.bind({});
-TopPart.args = {
-  topPart: true,
+Default.args = {
+  topPart: false,
```
Comment (mJarsater) [inline tegel/src/components/footer/footer.stories.ts:89]: We could remove this and just pass a defaultValue to the initial props instead.

**Review by mJarsater — APPROVED** [review]
LGTM

File: tegel/src/components/footer/footer.stories.ts (hunk 1 above)
Comment (theJohnnyMe) [reply tegel/src/components/footer/footer.stories.ts:89]: I was thinking about it, but if we define defaultValue in args, we got warning in console that from version 7 of Storybook defaultValue won't be supported. Therefore I defined it for this story too, but we can explore for other ways too.

**Review by victorscania — APPROVED** [review]

**Review by EmelieLitwin — APPROVED** [review]
Good job! 😍

File: tegel/src/components/footer/footer.stories.ts (hunk 1 above)
Comment (mJarsater) [reply tegel/src/components/footer/footer.stories.ts:89]: Yep, read up on it here: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#no-longer-inferring-default-values-of-args

And we should maybe add a ticket for updating all "defaultValue" to 
```
table: {
dafaultValue: {}
}
```

Comment (theJohnnyMe) [reply tegel/src/components/footer/footer.stories.ts:89]: I agree, let me add then a ticket so this is standardised too then, good finding ;)

**Pull request merged**